tests/gtests/test_convolution_eltwise_forward_bf16.cpp

```diff
@@ -56,7 +56,8 @@ TEST_P(convolution_test, TestConvolutionEltwise)
     EXPAND_ARGS(PARAMS_CONV(eltwise_soft_relu, __VA_ARGS__)), \
     EXPAND_ARGS(PARAMS_CONV(eltwise_logistic, __VA_ARGS__)), \
     EXPAND_ARGS(PARAMS_CONV(eltwise_exp, __VA_ARGS__)), \
-    EXPAND_ARGS(PARAMS_CONV(eltwise_swish, __VA_ARGS__))
+    EXPAND_ARGS(PARAMS_CONV(eltwise_swish, __VA_ARGS__)), \
+    EXPAND_ARGS(PARAMS_CONV(eltwise_mish, __VA_ARGS__))
```

Comment:
I would prefer not to touch mkldnn tests at all. Lets use only InferenceEngine tests for validation.

Reply:
Done